stl/inc/format

```diff
@@ -1116,6 +1116,41 @@ _OutputIt _Write(_OutputIt _Out, monostate) {
     return _Out;
 }
 
+// clang-format off
+template <class _CharT, class _OutputIt, class _Arithmetic>
+    requires is_arithmetic_v<_Arithmetic> && !same_as<_Arithmetic, bool> && !same_as<_Arithmetic, _CharT>

```

Comment:
The grammar for the expression in a _requires-clause_ is restricted to `&&`, `||`, and _primary-expression_ to avoid parsing ambiguities; anything else requires using `parens` since `(` _expression_ `)` is a _primary-expression_:
```suggestion
    requires (is_arithmetic_v<_Arithmetic> && !same_as<_Arithmetic, bool> && !same_as<_Arithmetic, _CharT>)
```